Source/Other/CategoryAdditions/SPDateAdditions.h

```diff
@@ -32,6 +32,7 @@
 
 + (uint64_t)monotonicTime;
 + (NSTimeInterval)timeIntervalSinceMonotonicTime:(uint64_t)comparisonTime;
+#ifdef DEBUG
 -(NSString *)formattedDateWithFormat:(NSString *)format timeZone:(NSTimeZone *)timeZone locale:(NSLocale *)locale;
```

Comment:
can we get rid of that ? I know you won§t have the tests, but you already have them :)

Reply:
can do